reference-implementation/lib/transform-stream.js

```diff
@@ -97,40 +105,51 @@ function TransformStreamErrorInternal(transformStream, e) {
   }
 }
 
-function TransformStreamReadyPromise(transformStream) {
-  assert(transformStream._backpressureChangePromise !== undefined);
+// Used for preventing the next write() call on TransformStreamSink until there
+// is no longer backpressure.
+function TransformStreamBackpressureGonePromise(transformStream) {
```

Comment:
I don't see how this name change makes it clearer. ReadyPromise is analogous to the [[readyPromise]] that writablestream writers have.